contrib/chart/backstage/templates/backstage-auth-config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "backstage.fullname" . }}-auth
+data:
+  AUTH_GOOGLE_CLIENT_ID: {{ .Values.auth.google.clientId }}
```

Comment:
Might as well set these with the same method as the other config

Reply:
discussed on discord and agreed to keep them here since there is no good way today to move these to the yaml config file